telemetry/definitions/commonDefinitions.json

```diff
@@ -9,7 +9,14 @@
         {
             "name": "appRunnerServiceStatus",
             "type": "string",
-            "allowedValues": ["CREATE_FAILED", "RUNNING", "DELETED", "DELETE_FAILED", "PAUSED", "OPERATION_IN_PROGRESS"],
+            "allowedValues": [
```

Comment:
Was anything of note added to this file or was it just reformatted?

Reply:
This file got re-formatted - I think by the git commit hook. Should I try to revert it?

Reply:
Up to you.  I went through it and didn't see any content changes, just formatting.  It's a little weird in that it went both from horizontal -> vertical and vertical -> horizontal, though I'd say the reformat is probably better because it made short runs of text horizontal and long runs vertical.  The concern I suppose would be if anyone else had a hook/IDE that would automatically reformat it back to what it was and cause thrashing.

Reply:
I am reverting this file and excluding it from prettier hooks. I think this repo would only get the hooks applied if you're working in the `lsp` folder and run `npm install`, so formatting that file can be left for another time.